build.sbt

```diff
@@ -80,3 +94,21 @@ fork in Test := true
 
 // forward stdin/out to fork, so the OpenIE CLI can be run in sbt.
 connectInput in run := true
+
+// The OpenIE codebase has many insignificant style warnings that appear
+// whenever the code is compiled. There are also some important-sounding
+// warnings which don't seem to affect the behavior but should be
+// fixed eventually:
+//
+//   Classes must override both equals and hashCode, not just one
```

Comment:
This almost always produces bugs in the resultant code; it's not merely bad style. This should not be suppressed by disabling compiler warnings.